src/screencast/screencast_common.c

```diff
@@ -23,13 +23,18 @@ void randname(char *buf) {
 	}
 }
 
-static char *gbm_find_render_node() {
+static char *gbm_find_render_node(drmDevicePtr device) {
```

Comment:
Nit: in general I advise against using libdrm's `Ptr` types as they are confusing for no good reason. `drmDevice *device` is less confusing.

Reply:
done